<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>QgsAttributeDialogBase</class>
 <widget class="QDialog" name="QgsAttributeDialogBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>267</width>
    <height>286</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Enter Attribute Values</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>10</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>93</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mOkButton" >
       <property name="text" >
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mCancelButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="0" >
    <widget class="Q3Table" name="mTable" >
     <property name="numRows" >
      <number>0</number>
     </property>
     <property name="numCols" >
      <number>2</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3Table</class>
   <extends></extends>
   <header>q3table.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
